src/test/java/com/google/api/codegen/testsrc/common/common_resources.proto

```diff
@@ -0,0 +1,68 @@
+// Copyright 2019 Google LLC.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This file contains stub messages for common resources in GCP.
+// It is not intended to be directly generated, and is instead used by
+// other tooling to be able to match common resource patterns.
+syntax = "proto3";
+
+package google.cloud;
```

Comment:
This doesn't have any of the language pkg options. Are those necessary?

Reply:
This is the `commor_resources.proto` file, same as at https://github.com/googleapis/googleapis/blob/master/google/cloud/common_resources.proto. 
Here is is used only for tests (as from point of view of gapic-generator, it is just a regular proto file, gapic-generator is technically not aware of common_resources concept). 
In prod environment the file from googleapis will be used.

I guess the file does not have the other options because it simply does not need them (even the messages are fake, they exist only to declare annoations in them).